src/Miki/Modules/DeveloperModule.cs

```diff
@@ -339,5 +340,44 @@ public async Task BanUserAsync(IContext e)
 					.BanAsync(context);
 			}
 		}
-	}
+
+        [Command("dailyedit")]
+        [RequiresScope("developer")]
+        public class DailyEditCommand
+        {
+            [Command("resettimer")]
+            [RequiresScope("developer")]
+            public async Task ResetDailyAsync(IContext e)
+            {
+                MikiDbContext context = e.GetService<MikiDbContext>();
```

Comment:
If it is possible, avoid using raw database handling inside of the business context. I know it's still very much used everywhere, but there's an effort to no longer do this.